tests/mocks/footer.ts

```diff
@@ -0,0 +1,44 @@
+const footerLinks = [
```

Comment:
I am a bit unclear why we need to mock this while they don't come via an API and they just get rendered by Frontend?